src/components/HeaderTop/HeaderTop.js

```diff
@@ -1,13 +1,49 @@
+import { useNavigate } from 'react-router-dom';
 import './HeaderTop.scss';
-const HeaderTop = ({ loged }) => {
+
+const HeaderTop = ({ loged, setLoged }) => {
+  const navigate = useNavigate;
+
+  const onClickHeaderLogin = () => {
+    if (loged == true) {
```

Comment:
- `isLoggedIn`과 같이 명확하게 작성하면 좋을 것 같습니다.
- isLoggedIn 자체가 boolean이기 때문에, true/false와 동일한지 비교할 필요 없이 isLoggedIn을 조건으로 사용하시면 됩니다.
```suggestion
    if (isLoggedIn) {
```

Reply:
그리고 로그인 / 로그아웃 유무는 이 프로젝트에서는 브라우저 스토리지(localStorage)에 있는 토큰의 유무로 결정할 것 같은데,
그렇다면 로그아웃할 때 굳이 백엔드에 요청을 할 필요가 없습니다. 실제로는 더 복잡하게 구현될 수도 있지만, 단순히 localStorage에 저장된 토큰을 삭제하시면 됩니다.